osfclient/models/storage.py

```diff
@@ -48,7 +49,7 @@ def files(self):
         return self._iter_children(self._files_url, 'file', File,
                                    self._files_key)
 
-    def create_file(self, path, fp, update=False):
+    def create_file(self, path, fp, force=False, update=False):
```

Comment:
Can you extend the doc string and point out how force and update work together/are different from each other.

Reply:
Right, forgot to update the doc string, I'll get right on that.